pyDeltaRCM/default.yml

```diff
@@ -139,3 +139,9 @@ coeff_U_ero_sand:
 alpha:
   type: ['float', 'int']
   default: 0.1
+checkpoint:
```

Comment:
how about renaming as `save_checkpoint`, so that it is consistent with the other "save" flags above. I think you can logically "group" the param with those params in this list, too.

Reply:
Yes, good idea.